VLCAlertViewController.swift

```diff
@@ -0,0 +1,58 @@
+/*****************************************************************************
+ * VLCAlertControllerExtension.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Quentin Richard <Quentinr75@gmail.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import UIKit
+
+typealias AlertAction = (UIAlertAction) -> Void
+
+
+@objcMembers class ButtonAction: NSObject {
+    let buttonTitle: String
+    var buttonAction: AlertAction
+    init(buttonTitle: String, buttonAction: @escaping AlertAction) {
+        self.buttonTitle = buttonTitle
+        self.buttonAction = buttonAction
+    }
+
+}
+
+@objc class VLCAlertViewController: UIAlertController {
+
+   @objc class func alertViewManager(title: String, errorMessage: String? = nil, viewController: UIViewController,
```

Comment:
The function name is misleading for me. We should describe what the function does. If I understand the code correctly it creates an alert with a title from a Viewcontroller that is passed in. Correct?